src/OnboardingSPA/pages/Steps/GetStarted/Welcome/tabContent.json

```diff
@@ -4,21 +4,23 @@
             "name": "tab1",
             "title": "YOUR CONTENT",
             "subtitle": "Publish boldly with WordPress Blocks.",
-            "text": "Build a beautiful %s using a visual builder. Block Patterns accelerate layout assembly and storytelling with premade designs.",
+            "text": "Build a beautiful %s using a visual builder. Block Patterns accelerate telling your story or tending your store with professional designs.",
+            "buttonText": "Try",
+            "buttonUrl": "",
```

Comment:
Do we need this?

Reply:
Ah, didn't mean this one to go up, I was moving quick through history. Tried some call-to-action button treatments but it was too cluttered.